### ISSUE TITLE
Logitech Lift vertical mouse's button is incorrectly swapped

### ISSUE THREAD

**Issue by soyflourbread** [systemd/systemd#26676]
### systemd version the issue has been seen with

systemd 253.1-1 [core] (x86_64)

### Used distribution

Arch Linux

### Linux kernel version used

6.2.2-zen1-1-zen

### CPU architectures issue was seen on

x86_64

### Component

systemd-hwdb

### Expected behaviour you didn't see

For a right-handed mouse, the left button is mapped to primary button and the right button is mapped to secondary.

Upgrading to systemd 253.1 introduces this problem,
likely a regression from #26523.

### Unexpected behaviour you saw

Left button is mapped to secondary button, etc.

### Steps to reproduce the problem

Have a right-handed Logitech Lift, and connect it to the computer through Logitech Bolt receiver. The left-right buttons will be swapped.

### Additional program output to the terminal or log subsystem illustrating the issue

```sh
$> udevadm info /dev/input/mouse0 | cat

P: /devices/pci0000:00/0000:00:08.1/0000:0e:00.3/usb2/2-2/2-2:1.1/0003:046D:C548.0002/input/input4/mouse0
M: mouse0
R: 0
U: input
D: c 13:32
N: input/mouse0
L: 0
S: input/by-id/usb-Logitech_USB_Receiver-if01-mouse
S: input/by-path/pci-0000:0e:00.3-usb-0:2:1.1-mouse
E: DEVPATH=/devices/pci0000:00/0000:00:08.1/0000:0e:00.3/usb2/2-2/2-2:1.1/0003:046D:C548.0002/input/input4/mouse0
E: DEVNAME=/dev/input/mouse0
E: MAJOR=13
E: MINOR=32
E: SUBSYSTEM=input
E: USEC_INITIALIZED=8975258
E: ID_INPUT=1
E: ID_INPUT_MOUSE=1
E: ID_BUS=usb
E: ID_MODEL=USB_Receiver
E: ID_MODEL_ENC=USB\x20Receiver
E: ID_MODEL_ID=c548
E: ID_SERIAL=Logitech_USB_Receiver
E: ID_VENDOR=Logitech
E: ID_VENDOR_ENC=Logitech
E: ID_VENDOR_ID=046d
E: ID_REVISION=0501
E: ID_TYPE=hid
E: ID_USB_MODEL=USB_Receiver
E: ID_USB_MODEL_ENC=USB\x20Receiver
E: ID_USB_MODEL_ID=c548
E: ID_USB_SERIAL=Logitech_USB_Receiver
E: ID_USB_VENDOR=Logitech
E: ID_USB_VENDOR_ENC=Logitech
E: ID_USB_VENDOR_ID=046d
E: ID_USB_REVISION=0501
E: ID_USB_TYPE=hid
E: ID_USB_INTERFACES=:030101:030102:030000:
E: ID_USB_INTERFACE_NUM=01
E: ID_USB_DRIVER=usbhid
E: ID_PATH=pci-0000:0e:00.3-usb-0:2:1.1
E: ID_PATH_TAG=pci-0000_0e_00_3-usb-0_2_1_1
E: DEVLINKS=/dev/input/by-id/usb-Logitech_USB_Receiver-if01-mouse /dev/input/by-path/pci-0000:0e:00.3-usb-0:2:1.1-mouse
```

**Comment by yuwata** [comment]
Duplicate of #26671.

**Comment by yuwata** [comment]
Fix is waiting in #26677.
